extensions/wikia/CommunityPage/models/CommunityPageSpecialHelpModel.class.php

```diff
@@ -4,12 +4,14 @@ class CommunityPageSpecialHelpModel {
 	public function getData() {
 		return [
 			'title' => wfMessage( 'communitypage-help-module-title' )->plain(),
+			'titleIcon' => file_get_contents( 'extensions/wikia/CommunityPage/images/help_icon.svg' ),
```

Comment:
`DesignSystemHelper` can be only used to get svg's uploaded to Design System. This icon is not there